packages/sdk/identity/src/odis/quota.ts

```diff
@@ -0,0 +1,35 @@
+import { Address } from '@celo/base'
+import {
+  CombinerEndpoint,
+  PnpQuotaRequest,
+  PnpQuotaResponse,
+} from '@celo/phone-number-privacy-common'
+import { AuthSigner, ErrorMessages, queryOdis, ServiceContext } from './query'
+
+export async function getQuotaStatus(
+  account: Address,
+  signer: AuthSigner,
+  context: ServiceContext,
+  clientVersion?: string,
+  sessionID?: string
+): Promise<number> {
+  const body: PnpQuotaRequest = {
+    account,
+    version: clientVersion ? clientVersion : 'unknown',
+    authenticationMethod: signer.authenticationMethod,
+    sessionID,
+  }
+
+  const response = await queryOdis<PnpQuotaResponse>(
```

Comment:
Could you also update queryOdis to take in a request generic instead? (same pattern as sendOdisDomainRequest below)

Reply:
updated!